godot-core/src/builtin/quaternion.rs

```diff
@@ -32,8 +32,11 @@ impl Quaternion {
 
     pub fn from_angle_axis(axis: Vector3, angle: real) -> Self {
         let d = axis.length();
-        if d == 0.0 {
-            Self::new(0.0, 0.0, 0.0, 0.0)
+        // This is consistent with godot.
+        // We could normalize the vector if it isn't.
+        // But godot doesn't. So we don't.
```

Comment:
[Comment width](https://github.com/godot-rust/gdext/pull/561#discussion_r1451568195) + capitalization of "Godot" 🙂

Reply:
Aha, forgot about the too short bound :smile: